internal/chmod_sock.go

```diff
@@ -0,0 +1,19 @@
+package internal
+
+import (
+	"log"
+	"os"
+	"os/exec"
+)
+
+func CheckSockFile(sockDirName, sockFileName string) {
+	if _, err := os.Stat(sockFileName); err != nil {
+		return
+	}
+	if _, err := exec.Command("chmod", "777", sockDirName).CombinedOutput(); err != nil {
```

Comment:
```suggestion
	if _, err := exec.Command("chmod", "775", sockDirName).CombinedOutput(); err != nil {
```
In this case watch mechanism is not needed anymore

Reply:
Done